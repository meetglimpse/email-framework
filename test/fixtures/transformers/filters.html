<!-- Append -->
<div append="ing append">test</div>
<!-- Prepend -->
<div prepend="test">ing prepend</div>

<!-- Uppercase -->
<div uppercase>test</div>
<!-- Lowercase -->
<div lowercase>TEST</div>
<!-- Capitalize -->
<div capitalize>test</div>

<!-- Ceil -->
<div ceil>1.2</div>
<!-- Floor -->
<div floor>1.2</div>
<!-- Round -->
<div round>1234.567</div>

<!-- Escape -->
<div escape>"&'<></div>
<!-- Escape Once -->
<div escape-once>1 &lt; 2 &amp; 3</div>

<!-- lstrip -->
<div lstrip>   test   </div>
<!-- rstrip -->
<div rstrip>   test   </div>
<!-- trim -->
<div trim>   test   </div>

<!-- Minus -->
<div minus="2">3.02</div>
<!-- Plus -->
<div plus="2">3.02</div>
<!-- Times -->
<div times="2">6.04</div>
<!-- Divide -->
<div divide-by="2">12.08</div>
<!-- Modulo -->
<div modulo="2">3</div>

<!-- Newline to br -->
<div newline-to-br>
  test
  test
</div>
<!-- Strip newlines -->
<div strip-newlines>
  test
  test
</div>

<!-- Remove -->
<div remove="rain">I strained to see the train through the rain</div>
<!-- Remove First -->
<div remove-first="rain">I strained to see the train through the rain</div>

<!-- Replace -->
<div replace="t|test">test</div>
<!-- Replace First -->
<div replace-first="t|test">test</div>

<!-- Size -->
<div size>This string is 33 characters long</div>

<!-- Slice -->
<div slice="1">test</div>
<!-- Slice with endIndex -->
<div slice="0,-1">test</div>

<!-- Truncate -->
<div truncate="17">Ground control to Major Tom.</div>
<!-- Truncate (do nothing) -->
<div truncate="17">Ground control to</div>
<!-- Truncate with custom ellipsis -->
<div truncate="17, no one">Ground control to Major Tom.</div>

<!-- Truncate words -->
<div truncate-words="2">Ground control to Major Tom.</div>
<!-- Truncate words with custom ellipsis -->
<div truncate-words="2, over and out">Ground control to Major Tom.</div>

<!-- URL decode -->
<div url-decode>%27Stop%21%27+said+Fred</div>
<!-- URL encode -->
<div url-encode>user@example.com</div>
