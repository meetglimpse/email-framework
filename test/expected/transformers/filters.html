<!-- Append -->
<div>testing append</div>
<!-- Prepend -->
<div>testing prepend</div>

<!-- Uppercase -->
<div>TEST</div>
<!-- Lowercase -->
<div>test</div>
<!-- Capitalize -->
<div>Test</div>

<!-- Ceil -->
<div>2</div>
<!-- Floor -->
<div>1</div>
<!-- Round -->
<div>1235</div>

<!-- Escape -->
<div>&#34;&amp;&#39;&lt;&gt;</div>
<!-- Escape Once -->
<div>1 &lt; 2 &amp; 3</div>

<!-- lstrip -->
<div>test   </div>
<!-- rstrip -->
<div>   test</div>
<!-- trim -->
<div>test</div>

<!-- Minus -->
<div>1.02</div>
<!-- Plus -->
<div>5.02</div>
<!-- Times -->
<div>12.08</div>
<!-- Divide -->
<div>6.04</div>
<!-- Modulo -->
<div>1</div>

<!-- Newline to br -->
<div><br>  test<br>  test<br></div>
<!-- Strip newlines -->
<div>  test  test</div>

<!-- Remove -->
<div>I sted to see the t through the </div>
<!-- Remove First -->
<div>I sted to see the train through the rain</div>

<!-- Replace -->
<div>testestest</div>
<!-- Replace First -->
<div>testest</div>

<!-- Size -->
<div>33</div>

<!-- Slice -->
<div>est</div>
<!-- Slice with endIndex -->
<div>tes</div>

<!-- Truncate -->
<div>Ground control to...</div>
<!-- Truncate (do nothing) -->
<div>Ground control to</div>
<!-- Truncate with custom ellipsis -->
<div>Ground control to no one</div>

<!-- Truncate words -->
<div>Ground control...</div>
<!-- Truncate words with custom ellipsis -->
<div>Ground control over and out</div>

<!-- URL decode -->
<div>'Stop!' said Fred</div>
<!-- URL encode -->
<div>user%40example.com</div>
